pkg/arvo/app/chat-store.hoon

```diff
@@ -194,10 +194,9 @@
   =:  length.config.u.mailbox  +(length.config.u.mailbox)
       number.envelope.act  length.config.u.mailbox
       envelopes.u.mailbox  (snoc envelopes.u.mailbox envelope.act)
```

Comment:
To be clear, envelopes are zero-indexed, correct? (Ie, the number of the latest envelope in a mailbox is always one one less than the length of the mailbox.)

FWIW, I would be inclined to store the envelopes reversed, both for easy update (cells cons instead of `+snoc` append which traverses the whole list) and because you're more likely to read recent messages. I should've picked up on this in the original review. It's probably not worth changing OTA, but worth considering in the future.

Reply:
No, it's 1-indexed. It turns out the change over to =: messed up quite a few things. I've now changed this to increment as well.

Reply:
I don't know. Maybe. Seems reasonable.

Reply:
I'd argue that it should be 0-indexed (for consistency with our list gates if nothing else), but this is also not the time or place. Sorry for the insufficiency of my earlier review!